Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/golang:alpine AS builder
+
+RUN apk update && \
+    apk add --no-cache gcc libc-dev
+
+WORKDIR /app
+
+COPY . .
+
+RUN CGO_ENABLED=1 go build -ldflags="-s -w -extldflags '-static'" -tags netgo -installsuffix netgo  -o goauth /app/cmd/goauth
+
+FROM scratch
+
+WORKDIR /app
+
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```

Comment:
IMHO not relevant as we don't spawn any kind of ssl context

Reply:
Without this line the email notification for registering is not working.

Reply:
Ok, but add some kind of comment to indicate the usage